### PR TITLE
Add tokens to cloudformation actor.

### PR DESCRIPTION
**Pull request by swaggy**
@diranged 

Also removed some redundant parens :)

### CODE REVIEW

File: kingpin/actors/aws/cloudformation.py [hunk 1]
```diff
@@ -506,6 +506,9 @@ class Create(CloudFormationBaseActor):
         'name': (str, REQUIRED, 'Name of the stack'),
         'parameters': (ParametersConfig, {}, 'Parameters passed into the CF '
                                              'template execution'),
+        'tokens': (dict, {},
```
Comment (diranged) [inline kingpin/actors/aws/cloudformation.py]: Missing documentation

File: kingpin/actors/aws/cloudformation.py [hunk 2]
```diff
@@ -688,6 +692,9 @@ class Stack(CloudFormationBaseActor):
                        'Action to take if the stack fails to be created'),
         'parameters': (ParametersConfig, {}, 'Parameters passed into the CF '
                                              'template execution'),
+        'tokens': (dict, {},
```
Comment (diranged) [inline kingpin/actors/aws/cloudformation.py]: Missing docs

**Comment by swaggy** [comment]
I see you're going through and pruning old PRs. I don't recall the motivation for this so feel free to close it.

**Comment by diranged** [comment]
I vaguely remember -- but now that we support defaults in the CF Parameters, this doesn't make much sense. Good CF stacks take params, not tokens..

**Pull request closed without merging**